packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -354,6 +364,15 @@ export default class RunPanelStore {
                 break;
             }
         }
+
+        this.statistics = {
```

Comment:
Is this required? We're decorating the `statistics` object observable from the start, and we're not changing any of its properties, so it should automatically catch onto any changes made. Please check.

- https://mobx.js.org/refguide/object.html

Reply:
the `reaction` could not understand the update and made no change in sessionStorage, so I had to do that.